account_invoice_line_sort/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2015 Acsone
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': "Sort Customer Invoice Lines",
+
+    'summary': """
+        Manage sort of customer invoice lines by customers""",
+
+    'author': "ACSONE SA/NV,Odoo Community Association (OCA)",
+    'website': "http://acsone.eu",
```

Comment:
@cristinamartinrod  Replace URL as per [this](https://agithub.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)